libraries/rush-lib/src/logic/EventHooksManager.ts

```diff
@@ -53,7 +53,7 @@ export class EventHooksManager {
           console.error(
```

Comment:
https://github.com/microsoft/rushstack/blob/bfff5067cd17bb6eeb4ba6cc40bff3a020f93ccb/libraries/rush-lib/src/logic/EventHooksManager.ts#L23-L31

Does this `console.log()` get printed when  `process.env.INSIDERUSH==1`?  That might be misleading

Reply:
Added a check for then environment cue so this doesn't get printed.